### DISCUSSION TITLE
HIP SDK 5.5 for Windows

### DISCUSSION THREAD

**Discussion by saadrahim** [ROCm/ROCm#2347] [Announcements]
AMD is pleased to announce the availability of the HIP SDK for Windows as part of the ROCm platform. The [HIP SDK OS and GPU support page](https://rocm.docs.amd.com/en/docs-5.5.1/release/windows_support.html) lists the versions of Windows and GPUs validated by AMD. HIP SDK features on Windows are described in detail in our [What is ROCm?](https://rocm.docs.amd.com/en/docs-5.5.1/rocm.html#rocm-on-windows) page and differs from the Linux feature set. Visit [Quick Start](https://rocm.docs.amd.com/en/docs-5.5.1/deploy/windows/quick_start.html#) page to get started. Known issues are tracked on [GitHub](https://github.com/RadeonOpenCompute/ROCm/issues?q=is%3Aopen+label%3A5.5.1+label%3A%22Verified+Issue%22+label%3AWindows).

<hr /><em>This discussion was created from the release <a href='https://github.com/RadeonOpenCompute/ROCm/releases/tag/win-5.5'>HIP SDK 5.5 for Windows</a>.</em>

**Comment by johnnynunez** [comment]
congrats! so, when will there be pytorch support for windows? @saadrahim

**Reply by saadrahim** [reply]
No comment on future release plans.

From a technical standpoint, there are a few PyTorch build dependencies that need enablement on Windows such as MIOpen. Until the prerequisites are ported to Windows, pytorch support will not be possible.

**Reply by johnnynunez** [reply]
why 5.5.1 released and not 5.6?

**Reply by nyanmisaka** [reply]
“enable MIOpen on Windows” PR - https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2272

**Reply by johnnynunez** [reply]
uoooh

**Reply by heloess** [reply]
If we do not able to use AI frameworks on Windows what is the point then?

**Reply by johnnynunez** [reply]
blender, Houdini etc it's working now

**Reply by johnnynunez** [reply]
míopen I think that it's like cudnn

**Comment by YellowRoseCx** [comment]
Awesome!

**Comment by yamfun** [comment]
So we can Stable Diffusion on Windows now with ROCm instead of DirectML?

**Reply by saadrahim** [reply]
The HIP SDK launched on Windows today does not enable AI frameworks.

**Reply by yamfun** [reply]
I regret my choice of AMD everyday since I play with SD 😭

**Reply by YellowRoseCx** [reply]
Is hipBLAS an AI framework (or would that be PyTorch?)? I saw the hipBLAS files in the windows install, would rocBLAS work on windows with a 6800 XT? 
Thanks again for the updates, AMD is awesome

**Reply by johnnynunez** [reply]
“enable MIOpen on Windows” PR - https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2272 @YellowRoseCx

**Comment by mlv-** [comment]
Any chance to see an expansion in official support of GPUs on linux like we're seeing on windows?

**Comment by PennyFranklin** [comment]
Glad to see that. Hope that  the whole  rocm platform support come soon. Can't wait to use my 79xtx to run  sdwebui on win11.

**Comment by HGGshiwo** [comment]
Does ROCm support wsl?

**Reply by saadrahim** [reply]
No, ROCm does not support wsl.

**Comment by BahzBeih** [comment]
hope you add AI frameworks on Windows support so we can use stable diffusion webui in windows 🤞🤞🤞🤞🤞
